README.md

```diff
@@ -15,17 +14,22 @@ This is a Scala/Play frontend to allow Partnerships to provide their information
    `sbt 'run 9722 -Dapplication.router=testOnlyDoNotUseInAppConf.Routes'`
 
 ### Testing
+
```

Comment:
Were all these formatting changes intentional on the readme?

Reply:
No they weren't, they're the result of auto formatting